go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/hashicorp/raft
+
+go 1.12
+
+require (
+	github.com/armon/go-metrics v0.0.0-20190430140413-ec5e00d3c878
+	github.com/boltdb/bolt v1.3.1 // indirect
+	github.com/hashicorp/go-hclog v0.9.0
+	github.com/hashicorp/go-msgpack v0.5.4
```

Comment:
go-msgpack has had a release since this PR, might want to re-init so it pulls in the latest version

Reply:
Thanks!